<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1393</width>
    <height>785</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>10.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>63</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QCheckBox" name="tranCheck_2">
       <property name="text">
        <string>Translação</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="escCheck_2">
       <property name="text">
        <string>Escala</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="rotCheck_2">
       <property name="text">
        <string>Rotação</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="refCheck_2">
       <property name="text">
        <string>Reflexão</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QCheckBox" name="cisCheck_2">
       <property name="text">
        <string>Cisalhamento</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="apButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Aplicar</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>201</width>
      <height>112</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="xLab_2">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>x:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="xSlid_2">
       <property name="cursor">
        <cursorShape>UpArrowCursor</cursorShape>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="yLab_2">
       <property name="text">
        <string>y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="ySlid_2">
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="my_qlabel" name="gridLab">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>1161</width>
      <height>761</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>201</width>
      <height>59</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="angLab_2">
       <property name="text">
        <string>Ângulo:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="angSlid_2">
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="singleStep">
        <number>15</number>
       </property>
       <property name="pageStep">
        <number>16</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSplitter" name="passosSplitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="passosLab_2">
     <property name="text">
      <string>Número de Passos:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nPassos_2"/>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="imgButton">
     <property name="text">
      <string>Imagem</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>X = 0, Y = 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1393</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>my_qlabel</class>
   <extends>QLabel</extends>
   <header location="global">my_qlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>rotCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>angLab_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>14</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>angSlid_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>xLab_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>16</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>xSlid_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>yLab_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>24</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>ySlid_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>24</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cisCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>angLab_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cisCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>angSlid_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cisCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>xLab_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cisCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>xSlid_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cisCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>yLab_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cisCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>ySlid_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>37</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>escCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>angLab_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>escCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>angSlid_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>escCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>xLab_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>escCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>xSlid_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>escCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>yLab_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>escCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>ySlid_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tranCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>angLab_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tranCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>angSlid_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tranCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>xLab_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tranCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>xSlid_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tranCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>yLab_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tranCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>ySlid_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>angLab_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>xLab_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>angSlid_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>xSlid_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>yLab_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refCheck_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>ySlid_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
